tox.ini

```diff
@@ -56,6 +56,25 @@ known_wagtail=wagtail
 default_section=THIRDPARTY
 sections=FUTURE,STDLIB,DJANGO,WAGTAIL,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
 
+[testenv:interactive]
+basepython=python3.9
+
+deps=
+    Django>=3.2,<3.3
```

Comment:
This shouldn't technically be needed, but I ran into an issue where tox doesn't seem to run the `install_command` unless there is _some_ dependency defined for the environment (try removing this section and see what happens).